addons/account/tests/test_chart_template.py

```diff
@@ -123,7 +122,7 @@ def _tax_vals(name, amount, tax_tag_id=None, children_tax_xmlids=None, active=Tr
 
 @tagged('post_install', '-at_install')
 @patch.object(AccountChartTemplate, '_get_chart_template_mapping', _get_chart_template_mapping)
-class TestChartTemplate(TransactionCase):
+class TestChartTemplate(BaseCommon):
```

Comment:
Done, mainly had to make it so we load the setUpClass of the parent of AccountTestInvoicingCommon (to avoid creating all the data of the class, but still use all the overriden methods